JotunnLib/Extensions/TransformExtensions.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Jotunn.Extensions
+{
+    public static class TransformExtensions
+    {
+        /// <summary>
+        ///     Extension method to find nested children by name using either
+        ///     a breadth-first or depth-first search. Default is breadth-first.
+        /// </summary>
+        /// <param name="transform"></param>
+        /// <param name="childName">Name of the child object to search for.</param>
+        /// <param name="breadthFirst"> Whether to preform a breadth first or depth first search. Default is breadth first.</param>
+        /// <returns></returns>
+        public static Transform FindDeepChild(
```

Comment:
As mentioned above, this method is already implemented in `Utils.FindChild` and should call the vanilla method